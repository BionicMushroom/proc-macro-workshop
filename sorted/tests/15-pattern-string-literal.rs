#[sorted::check]
fn main() {
    let x = "abc";

    #[sorted]
    match x {
        "def" => println!("def"),
        "abc def" => println!("abc def"),
        _ => println!("other"),
    }
}
