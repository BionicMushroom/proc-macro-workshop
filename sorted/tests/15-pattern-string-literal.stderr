error: `"abc def"` should sort before "def"
 --> tests/15-pattern-string-literal.rs:8:9
  |
8 |         "abc def" => println!("abc def"),
  |         ^^^^^^^^^
